src/Microsoft.Identity.Client/ApiConfig/AcquireTokenByRefreshTokenParameterBuilder.cs

```diff
@@ -36,8 +39,10 @@ namespace Microsoft.Identity.Client.ApiConfig
     public sealed class AcquireTokenByRefreshTokenParameterBuilder :
         AbstractClientAppBaseAcquireTokenParameterBuilder<AcquireTokenByRefreshTokenParameterBuilder>
     {
+        private AcquireTokenByRefreshTokenParameters Parameters { get; } = new AcquireTokenByRefreshTokenParameters();
```

Comment:
there are a lot of parameters now...could these be class specific, like "RefreshTokenParameters"...when you see Parameters later on, not clear what those parameters are referring to and how they are different from the commonParameters without looking more at the code. but maybe a non-issue.

Reply:
well...maybe not. i'm getting used to it now. 

---
In reply to: [248925442](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/798#discussion_r248925442) [](ancestors = 248925442)

Reply:
happy to talk on these further.

---
In reply to: [248925608](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/798#discussion_r248925608) [](ancestors = 248925608,248925442)